shared/cmd/flags.go

```diff
@@ -129,4 +129,9 @@ var (
 		Name:  "log-file",
 		Usage: "Specify log file name, relative or absolute",
 	}
+	// EnableUPnPFlag specifies if UPnP should be enabled or not. The default value is false.
+	EnableUPnPFlag = cli.BoolFlag{
+		Name:  "enable-upnp",
+		Usage: "Enable UPnP",
```

Comment:
please add a complete sentence for the `Usage`, similar to other usages

Reply:
Yes. please add a full description here.

Reply:
Done. Thanks.